velox/exec/WindowPartition.cpp

```diff
@@ -363,8 +363,11 @@ void WindowPartition::updateKRangeFrameBounds(
 
   vector_size_t start = 0;
   vector_size_t end;
+  // frameColumn is a column index into the original input rows, while
+  // orderByColumn is a column index into rows in data_ after the columns are
+  // reordered as per inputMapping_.
   RowColumn frameRowColumn = columns_[frameColumn];
-  RowColumn orderByRowColumn = columns_[inputMapping_[orderByColumn]];
+  RowColumn orderByRowColumn = data_->columnAt(orderByColumn);
```

Comment:
Ah I see... sortKeyInfo_[0] is from the re-ordered partition, so the lookup from inputMapping is not needed. Makes sense.